samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -493,32 +528,38 @@ uint8_t app_fp_adv_id_get(void)
 	return fp_adv_param.id;
 }
 
-int app_fp_adv_uninit(void)
+int app_fp_adv_init(void)
 {
 	int err;
+	int trigger_cnt;
 
-	app_fp_adv_mode_set(APP_FP_ADV_MODE_OFF);
-	app_fp_adv_rpa_rotation_suspend(false);
-
-	fp_adv_conn_clear();
-
-	err = fp_adv_set_teardown();
+	err = bt_fast_pair_fmdn_info_cb_register(&fmdn_info_cb);
 	if (err) {
-		LOG_ERR("Fast Pair: fp_adv_set_teardown failed (err %d)", err);
+		LOG_ERR("Fast Pair: bt_fast_pair_fmdn_info_cb_register returned error: %d", err);
 		return err;
 	}
 
+	STRUCT_SECTION_COUNT(app_fp_adv_trigger, &trigger_cnt);
+	if (trigger_cnt > BITS_PER_VAR(fp_adv_request_bm)) {
+		LOG_ERR("Fast Pair: invalid size of the advertising request bitmap");
+		return -ENOMEM;
+	}
```

Comment:
Consider making it an assert (it's actually static configuration issue + caller can do nothing about it).

Reply:
Seems resonable, changed.